cmd/grpcurl/grpcurl.go

```diff
@@ -30,13 +30,17 @@ import (
 	"github.com/fullstorydev/grpcurl"
 )
 
+var version = "<not set>"
```

Comment:
Maybe something a little more informative, like `dev build <no version set>`? And it would also be nice to update the `Makefile` so it uses `-ldflags` to set `main.version` to something like `dev build <git sha>` (though that's fine to omit from this PR and maybe just add a TODO; if you want to tackle in another branch, great, and if not I can address it).

Reply:
👍 Lets tackle it in this branch. I'll add it.